pinc/image_check.inc

```diff
@@ -1,6 +1,7 @@
 <?php
 
 $page_image_size_limit = 100; // kb
+$page_image_minimum_dimension = 5000; // pixels
```

Comment:
\<cough> ... 1000

Reply:
haha, I was testing the error messages and the project I had didn't have any small images so I had to bump the number :).